v3.0/reference/tidb-binlog/deploy.md

```diff
@@ -154,7 +154,11 @@ In environments of development, testing and production, the requirements on serv
 
 1. Obtain `initial_commit_ts`.
 
-    Run the following command to use `binlogctl` to generate the `tso` information which is needed for the initial start of Drainer:
+    In TiDB 3.0.6 or later versions, if the synchronization is started from the latest time point, you just need to set `initial_commit_ts` to `-1`. Otherwise, refer to the following methods to obtain a latest timestamp.
```

Comment:
```suggestion
    In TiDB 3.0.6 or later versions, if the replication is started from the latest time point, you just need to set `initial_commit_ts` to `-1`. Otherwise, refer to the following methods to obtain the latest timestamp.
```